docs/data/packages.json

```diff
@@ -1,12 +1,52 @@
 {
 	"v1": {
-		"libraries": ["fluid-framework", "@fluidframework/counter"],
-		"serviceClients": ["@fluidframework/azure-client", "@fluidframework/tinylicious-client"],
-		"utilities": ["@fluidframework/azure-service-utils"]
+		"Audience": {
```

Comment:
Let's be sure to add some notes about how this organization is maintained in the docs README.